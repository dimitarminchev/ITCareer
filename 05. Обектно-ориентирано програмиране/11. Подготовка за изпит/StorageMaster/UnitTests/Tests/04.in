RegisterStorage DistributionCenter VratsaDistribution
RegisterStorage Warehouse GoogleWarehouse
AddProduct Gpu 1200
AddProduct SolidStateDrive 205
AddProduct HardDrive 70
AddProduct HardDrive 120
SelectVehicle VratsaDistribution 0
LoadVehicle Gpu HardDrive
SendVehicleTo VratsaDistribution 0 GoogleWarehouse
UnloadVehicle GoogleWarehouse 3
END
