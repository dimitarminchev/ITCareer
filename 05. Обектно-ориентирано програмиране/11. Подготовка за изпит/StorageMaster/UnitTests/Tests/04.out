Registered VratsaDistribution
Registered GoogleWarehouse
Added Gpu to pool
Added SolidStateDrive to pool
Added HardDrive to pool
Added HardDrive to pool
Selected Van
Loaded 2/2 products into Van
Sent Van to GoogleWarehouse (slot 3)
Unloaded 2/2 products at GoogleWarehouse
GoogleWarehouse:
Storage worth: $1320.00
VratsaDistribution:
Storage worth: $0.00
